draft-irtf-cfrg-hpke.md

```diff
@@ -830,6 +829,20 @@ security models and assumptions, and do not consider attackers capable of quantu
 computation. A full proof of post-quantum security would need to take this
 difference into account, in addition to simply using a post-quantum KEM.
 
+## Pre-Shared Key Recommendations
+
+In the PSK mode (see {{mode-psk-auth}}), PSKs SHOULD be at least Nh bytes long,
+and SHOULD be derived from at least Nh bytes of entropy.  While an HPKE
+ciphertext does not reveal anything about the PSK (as a consequence of the
+IND-CCA2 property), application behaviors may create an oracle that tells an
```

Comment:
I prefer not to invoke IND-CCA here because IND-CCA does not state anything about if the ciphertext reveals smth about the key. It just assumes that the key is random and secret.

Reply:
That works!

Reply:
Actually IND-CCA is what makes the dictionary attack possible in case of a low-entropy key, because of the ciphertext integrity part of IND-CCA, which provides the (decryption failure) oracle.